resource-service/src/main/java/uk/gov/gchq/palisade/service/resource/model/ExceptionSource.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2020 Crown Copyright
```

Comment:
```suggestion
 * Copyright 2018-2021 Crown Copyright
```

Reply:
I think it should be left like this for now and then updated as part of PAL-569 .